fenixedu-ist-integration/src/main/java/pt/ist/fenixedu/integration/task/updateData/parking/SetDefaultRFIDForCarParkUsers.java

```diff
@@ -0,0 +1,48 @@
+package pt.ist.fenixedu.integration.task.updateData.parking;
+
+import java.util.stream.Stream;
+
+import org.fenixedu.academic.domain.Person;
+import org.fenixedu.bennu.core.domain.Bennu;
+import org.fenixedu.bennu.core.domain.User;
+import org.fenixedu.bennu.scheduler.annotation.Task;
+import org.fenixedu.bennu.scheduler.custom.CustomTask;
+import org.fenixedu.idcards.domain.SantanderCardInformation;
+
+import pt.ist.fenixedu.parking.domain.ParkingParty;
+
+@Task(englishTitle = "Set default RFID for car park users")
+public class SetDefaultRFIDForCarParkUsers extends CustomTask {
```

Comment:
Why is this a `CustomTask ` and not a `CronTask` it has an annotation and if I understand it correctly is intended to be scheduled. Also, `CustomTasks` don't go in the source code.